pkgs/applications/misc/notesnook/default.nix

```diff
@@ -1,12 +1,17 @@
 { lib, stdenv, appimageTools, fetchurl, _7zz }:
 
 let
-  pname = "notesnook";
-  version = "3.0.8";
-
   inherit (stdenv.hostPlatform) system;
   throwSystem = throw "Unsupported system: ${system}";
 
+  pname = "notesnook";
+  version = {
+    x86_64-linux = "3.0.13";
+    x86_64-darwin = "3.0.8";
+    aarch64-darwin = "3.0.8";
```

Comment:
Why `x86_64-linux` only but not the others?

Reply:
Hey @xzfc,
I actually don't own a Mac or ARM-based Linux system. So, I couldn't test them. I am new to NixOS and I don't know any other way to test them without actual MacBook or ARM-based Linux system

Reply:
We rarely use diverging versions of the same package for different platforms. As per [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#built-on-platforms), it's not required to test such changes on all platforms.

So, feel free to bump it for all platforms. Someone will check it sooner or later. To find hashes for other platforms you could use the command `nix store prefetch-file`.

Reply:
Oh sure. Will do that

Reply:
Done